dll/win32/comdlg32/lang/cdlg_Ro.rc

```diff
@@ -352,174 +352,174 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
+  LTEXT "Stâ&nga:", stc15, 88, 172, 30, 8
   EDITTEXT edt4, 119, 170, 36, 12, WS_TABSTOP|WS_GROUP|WS_BORDER
-  LTEXT "&Dreapta:", stc16, 159, 172, 30, 8
+  LTEXT "D&reapta:", stc16, 159, 172, 30, 8
   EDITTEXT edt6, 190, 170, 36, 12, WS_TABSTOP|WS_GROUP|WS_BORDER
   LTEXT "S&us:", stc17, 88, 192, 30, 8
   EDITTEXT edt5, 119, 190, 36, 12, WS_TABSTOP|WS_GROUP|WS_BORDER
-  LTEXT "J&os:", stc18, 159, 192, 30, 8
+  LTEXT "&Jos:", stc18, 159, 192, 30, 8
   EDITTEXT edt7, 190, 190, 36, 12, WS_TABSTOP|WS_GROUP|WS_BORDER
-  DEFPUSHBUTTON "Con&firmă", IDOK, 70, 220, 50, 14, BS_PUSHBUTTON
-  PUSHBUTTON "A&nulează", IDCANCEL, 126, 220, 50, 14
-  PUSHBUTTON "I&mprimantă…", psh3, 182, 220, 50, 14
+  DEFPUSHBUTTON "OK", IDOK, 70, 220, 50, 14, BS_PUSHBUTTON
+  PUSHBUTTON "Revocare", IDCANCEL, 126, 220, 50, 14
+  PUSHBUTTON "&Imprimantă…", psh3, 182, 220, 50, 14
 END
 
 NEWFILEOPENORD DIALOG 0, 0, 280, 164
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_VISIBLE | WS_POPUP | WS_CAPTION | WS_SYSMENU | WS_CLIPCHILDREN
-CAPTION "Deschide"
+CAPTION "Deschidere"
 FONT 8, "MS Shell Dlg"
 {
-    LTEXT	"&Caută în:",IDC_LOOKINSTATIC,4,6,43,8, SS_NOTIFY
+    LTEXT	"&Privire în:",IDC_LOOKINSTATIC,4,6,43,8, SS_NOTIFY
     COMBOBOX	IDC_LOOKIN,49,3,132,100,CBS_DROPDOWNLIST | CBS_OWNERDRAWFIXED | CBS_HASSTRINGS | WS_VSCROLL | WS_TABSTOP
 
     LTEXT	"" , IDC_TOOLBARSTATIC, 181, 2, 102, 17, NOT WS_GROUP | NOT WS_VISIBLE
     LISTBOX	IDC_SHELLSTATIC,4,20,272,85, LBS_SORT | LBS_NOINTEGRALHEIGHT | LBS_MULTICOLUMN | WS_HSCROLL | NOT WS_VISIBLE
 
-    LTEXT	"N&ume fișier:",IDC_FILENAMESTATIC,5,112,56,16, SS_NOTIFY
+    LTEXT	"&Nume fişier:",IDC_FILENAMESTATIC,5,112,56,16, SS_NOTIFY
     EDITTEXT	IDC_FILENAME,63,110,150,12,ES_AUTOHSCROLL
     CONTROL     "", cmb13, "ComboBoxEx32", CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_VSCROLL | WS_TABSTOP, 63,110,150,150
 
-    LTEXT	"Tip de &fișier:",IDC_FILETYPESTATIC,5,130,56,16, SS_NOTIFY
+    LTEXT	"&Fişiere de tipul:",IDC_FILETYPESTATIC,5,130,56,16, SS_NOTIFY
     COMBOBOX	IDC_FILETYPE,63,128,150,53,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
 
-    CONTROL	"Nem&odificabil",IDC_OPENREADONLY,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,63,148,150,10
+    CONTROL	"Desc&hidere doar în citire",IDC_OPENREADONLY,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,63,148,150,10
 
-    DEFPUSHBUTTON	"&Deschide",		 IDOK,222,110,54,14
-    PUSHBUTTON		"A&nulează",		 IDCANCEL,222,128,54,14
-    PUSHBUTTON		"&Manual…",		 pshHelp,222,145,54,14
+    DEFPUSHBUTTON	"&Deschidere",		 IDOK,222,110,54,14
+    PUSHBUTTON		"Revocare",		 IDCANCEL,222,128,54,14
+    PUSHBUTTON		"Aj&utor",		 pshHelp,222,145,54,14
 }
 
 NEWFILEOPENV2ORD DIALOG 0, 0, 370, 237
 STYLE DS_MODALFRAME | WS_VISIBLE | WS_POPUP | WS_CAPTION | WS_SYSMENU | WS_CLIPCHILDREN
-CAPTION "Deschide"
+CAPTION "Deschidere"
 FONT 8, "MS Shell Dlg"
 {
-    LTEXT "&Caută în:", IDC_LOOKINSTATIC, 28, 6, 43, 8, SS_NOTIFY
+    LTEXT "&Privire în:", IDC_LOOKINSTATIC, 28, 6, 43, 8, SS_NOTIFY
```

Comment:
Does that longer string still fit the 43? Please attach a picture for proof.

Reply:
It should.

![2024-04-30 00_32_57-Resource Hacker - comdlg32 dll mui](https://github.com/reactos/reactos/assets/113553683/00621df6-0930-42b8-a5c9-cee5087ee288)

Reply:
Resolved.